torchaudio/__init__.py

```diff
@@ -1,4 +1,6 @@
 from . import extension
+from . import prototype
```

Comment:
I think it's better not to import `prototype` automatically, so that users have to do explicit import by themselves.